Detectors/FIT/workflow/src/FT0ReconstructorSpec.cxx

```diff
@@ -35,25 +37,30 @@ void FT0ReconstructorDPL::run(ProcessingContext& pc)
   }
   mRecPoints.clear();
   auto digits = pc.inputs().get<const std::vector<o2::ft0::Digit>>("digits");
```

Comment:
why are you getting a vector here and not a span?

Reply:
This question was not answered

Reply:
@AllaMaevskaya , did you miss this?

Reply:
Sorry, I really misread question.
Digits are structure with event-wise data like BC, orbit and triggers and span of ChannelData, similar to ZDC as you requested. This Digits are collected in vector event-by-event. What is wrong?

Reply:
When you do ``pc.inputs().get<const std::vector...`` you create a local copy of the framework-owned data. This is a waste of the memory and CPU resources. Instead on can use span, which is essentially a pointer on these data. But using the span requires that elements of this span (Digits in your case) are copiable (do not contain pointers, vectors etc). So, the whole point of these ZDC-like structures was to make T0 objects copiable and use spans instead of the vectors. Just change ``auto digits = pc.inputs().get<const std::vector<o2::ft0::Digit>>("digits");`` to 
``auto digits = pc.inputs().get<gsl::span<o2::ft0::Digit>>("digits");``

Reply:
Done